roles/openshift_master_certificates/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
I'm cool with this, the approach would mirror what I've done with the `etcd_certificates` role.